ntc_templates/templates/cisco_ios_show_power_inline.textfsm

```diff
@@ -0,0 +1,28 @@
+Value Required INTERFACE (\S+)
+Value ADMIN (auto)
+Value OPER (on|off)
+Value POWER (\d+\.\d+)
+Value DEV_POWER (\d+\.\d+)
+Value DEVICE (.+?)
+Value CLASS (n\/a|\d)
+Value MAX (\d+\.\d)
+
+Start
+# Checking for modules headers
+  ^Module\s+Available\s+Used\s+Remaining\s*$$
+  ^Available:\d+(\.\d+)?\(w\)\s+Used:\d+(\.\d+)?\(w\)\s+Remaining:\d+(\.\d+)?\(w\)\s*$$
+  ^[\s(Wats)]*$$
+  ^[-\s]*$$
+  ^\d+\s+(n\/a|\d+\.\d+)\s+(n\/a|\d+\.\d+)\s+(n\/a|\d+\.\d+)\s*$$
+  ^Interface\s+Admin\s+Oper\s+Power(\(Watts\))?\s+Device\s+Class.*$$
+  ^\s+From PS\s+To Device\s*$$
+# actual data
+  ^${INTERFACE}\s+${ADMIN}\s+${OPER}\s+${POWER}\s+${DEVICE}\s+${CLASS}\s+${MAX}\s*$$ -> Record
+  ^${INTERFACE}\s+${ADMIN}\s+${OPER}\s+${POWER}\s+${DEV_POWER}\s+${DEVICE}\s+${CLASS}\s*$$ -> Record
+# empty lines
+  ^\s*$$
+# Optional footer
+  ^Totals:\s+\d+\s+(on|off)\s+\d+\.\d+\s+\d+\.\d+\s*$$
+# Command unknown
+  ^% Invalid command .*$$ -> Error
+  ^.*$$
```

Comment:
This "^.*$$" will capture anything/everything, you should try to end the state with "^. -> Error"

Quote form the README.md
"Each state should end with ^. -> Error. This helps to ensure we're accounting for every line within the raw output for the command. This doesn't mean we have to capture all the data as a Value, but we do have to account for it. In addition, it is also good to provide an expression to match blank lines, ^\s*$$"

Reply:
Template is updated as described